_i18n/es.yml

```diff
@@ -332,7 +358,7 @@ help_pages:
 
       Utilice esta sección de ayuda para conocer el proceso y los requisitos relacionados con la verificación de la identidad.
 
-      ## Solución de problemas frecuentes
+      ## Solución de problemas frecuentes {#help-question-list}
```

Comment:
WDYT of making this a class instead of an ID? In case we ever want more than one of these on a page?